themes/wporg-5ftf/functions.php

```diff
@@ -272,6 +272,10 @@ function get_badge_classes( $team ) {
 			$classes = array( 'badge-test-team', 'dashicons-desktop' );
 			break;
 
+		case 'themes team':
+			$classes = array( 'badge-themes', 'dashicons-admin-appearance' );
+			break;
```

Comment:
IIRC, the intention in #83 was to avoid a vague name like "themes", so that people wouldn't incorrectly assume that building their own themes was a 5ftF contribution.

So, I think the solution here might be to keep `themes team` removed, but update any data in the database to use  `theme review team` instead, and then prevent people from entering `themes team`.

I'll have to dig into it when we start the v2 milestone, though.